src/Route.jsx

```diff
@@ -1,3 +1,7 @@
+// https://deck.gl/#/documentation/deckgl-api-reference/layers/icon-layer
+// TODO:
+// use css transform to flip
```

Comment:
What do you mean by flip? What do you mean by CSS transform? Also we don't use inline comments for TODOs. Could you open an issue and then add a link to that issue here? `// TODO - {issue link}`